packages/block-editor/src/components/index.js

```diff
@@ -83,6 +83,7 @@ export {
 	RichTextShortcut,
 	RichTextToolbarButton,
 	__unstableRichTextInputEvent,
+	getRichTextValues as __unstableGetRichTextValues,
```

Comment:
We don't use unstable/experimental prefixes anymore, we prefer "private APIs"

Reply:
Had to revert in https://github.com/WordPress/gutenberg/pull/51201/commits/56fa206d66f8e3e8401b7124d8f43812b9263a7d because tests were failing and I don't understand why.